ci/rpm/build_url_repo.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+Creates a repository based on the BUILD_URL environmnent variable.
```

Comment:
This should be comment, not a command.

[2022-06-15T19:57:00.103Z] [91m/tmp/build_url_repo.sh: line 3: Creates: command not found

Reply:
How did that get through shellcheck?

Will be fixed in the next push.